quotient-reminder.c

```diff
@@ -1,17 +1,53 @@
-#include<stdio.h>
+/*
+Module: Compute-quotient-and-reminder.c
+
+Function:
+    Computing quotient and reminder of a given number.
+
+Copyright notice:
+    This file copyright (C) 2022 by
+    MCCI Corporation
+    3520 Krums Corners Road
+    Ithaca, NY 14850
+    An unpublished work. All rights reserved.
+    This file is proprietary information, and may not be disclosed or
+    copied without the prior permission of MCCI Corporation.
+
+Author:
+    Athisankar, MCCI Corporation   May 2022
+*/
+
+#include <stdio.h>
+
+/*********************************************\
+|
+|   Code.
+|
+\*********************************************/
+
 int main()
-{
-    int div,divisor,quot,rem;
+    {
+    printf("\n");
+    printf("*********************************************\n");
+    printf("THIS IS QUOTIENT and REMINDER v1.0.0\n");
+    printf("*********************************************\n");
+    printf("\n");
+    printf("IT IS USED TO FIND QUOTIENT AND REMINDER \n \n");
 
-    printf("enter div");
+    int div, divisor, quot, rem;
+
+    printf("ENTER DIV");
```

Comment:
term div is incomplete, what does it mean? In Division operation, there are two type of values involved.
1. Divident
2. Divisor